packages/teams-js/src/public/calendar.ts

```diff
@@ -84,7 +84,11 @@ export namespace calendar {
 
   /** Open calendar item parameters. */
   export interface OpenCalendarItemParams {
-    /** An unique base64-encoded string id that represents the event's unique identifier of the calendar item to be opened. */
+    /** An unique base64-encoded string id that represents the itemId or messageId.
+     * The only acceptable id for this itemId is the EWS (Exchange Web Services) ID. However,
```

Comment:
>EWS (Exchange Web Services) ID [](http://example.com/codeflow?start=53&length=30)

Is there a public documentation link that describes what the EWS ID is? Your explanation for how to create it from the graph ID is very helpful and probably sufficient but if we have a doc link that would be a good addition.